lib/ansible/plugins/action/__init__.py

```diff
@@ -747,7 +749,7 @@ def _low_level_execute_command(self, cmd, sudoable=True, in_data=None, executabl
         out = self._strip_success_message(out)
 
         display.debug("_low_level_execute_command() done: rc=%d, stdout=%s, stderr=%s" % (rc, stdout, stderr))
-        return dict(rc=rc, stdout=out, stdout_lines=out.splitlines(), stderr=err)
+        return dict(rc=rc, stdout=out, stdout_lines=out.splitlines(), stderr=err, stderr_lines=err.splitlines())
```

Comment:
We aren't guaranteed that the stderr coming back from a connection plugin is any particular type- in this case, it's coming back from a test as a bool. Recommend type-checking and supplying a single-element repr list if it's not a type that supports splitlines()